src/identities/CommandDiscover.ts

```diff
@@ -103,8 +103,8 @@ class CommandDiscover extends CommandPolykey {
                   case 'queued':
                     queuedSet.add(vertex);
                     break;
-                  case 'processed':
-                  case 'cancelled':
+                  case 'processed': // Fallthrough
+                  case 'cancelled': // Fallthrough
```

Comment:
Why need these comments?

Reply:
The basic linter (eslint) doesn't care, but IDEs (like WebStorm) actually report this as an error if there is a blank case without a fallthrough comment.